cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSIMongoSinkTest.java

```diff
@@ -0,0 +1,259 @@
+package com.telefonica.iot.cygnus.sinks;
```

Comment:
License header in new files

Reply:
Fixed in [25f98f6](https://github.com/telefonicaid/fiware-cygnus/pull/1818/commits/25f98f637db431e38bfee19706ae5ac09993eb24)